larva/src/main/java/org/frankframework/larva/LarvaConfig.java

```diff
@@ -43,7 +43,7 @@ public class LarvaConfig {
 	 * if allowReadlineSteps is set to true, actual results can be compared in line by using .readline steps.
 	 * Those results cannot be saved to the expected value defined inline, however.
 	 */
-	private @Getter @Setter boolean allowReadlineSteps = false;
+	private @Getter @Setter boolean allowReadlineSteps = true;
```

Comment:
Willen we dit?

Reply:
Omdat er een save knop zit in de larva en die gaat hierdoor kapot. Dan moet er iets komen dat er geen save knop is wanneer je deze property gebruikt (`a.b.c.readline=tekst`) of de property file zelf moet aangepast worden.

Reply:
Ja ik weet dat dit de "save" knop breekt, maar als mensen weten dat met "readline" dit niet werkt, dan is het misschien geen groot probleem ook als we die knop nog niet kunnen onderdrukken?

Voor het scenario wat ik had gemaakt, leek het mij handiger om geen extra files aan te hoeven maken voor een resultaat wat alleen maar een hele simpele string is.
Daarom had ik de readline-optie weer aangezet.

Reply:
Ik denk dat het handiger is dat we dat een andere keer oppakken, nu introduceren we 'vreemd' gedrag. We hebben hier geen/nauwelijks documentatie over, dan heb ik voor nu liever het omslachtige dan deze hidden feature met ongewenste side-effects.